Doc/library/logging.config.rst

```diff
@@ -126,6 +126,10 @@ in :mod:`logging` itself) and defining handlers which are declared either in
     .. versionadded:: 3.10
        The *encoding* parameter is added.
 
+    .. versionadded:: 3.11
```

Comment:
This is in the wrong place (`fileConfig` rather than `dictConfig`) and in any case should perhaps be in the [documentation on the details of the dictionary schema](https://docs.python.org/3/library/logging.config.html#dictionary-schema-details), in the `filters` bullet point.

Reply:
Indeed 🤦 .
I've added it to the section of user custom objects, have a look and let me know if you think it fits there.
Should we also add it to both the handlers and loggers section?